sdk/communication/communication-chat/src/chatThreadClient.ts

```diff
@@ -412,15 +422,15 @@ export class ChatThreadClient {
    * @param options - Operation options.
    */
   public async removeParticipant(
-    participant: CommunicationUserIdentifier,
+    participant: CommunicationIdentifier,
```

Comment:
Here it's right to take CommunicationIdentifier and not CommunicationIdentifierKind to make it easier on the developer so that they don't need to provide the kind property 👍